From 459cebc515b11fef06464b711b3b33cbc1dd8c26 Mon Sep 17 00:00:00 2001
From: pascalschumacher <pascalschumacher@gmx.net>
Date: Sun, 14 Jan 2018 11:45:10 +0100
Subject: [PATCH] IO-557: UnsupportedEncodingException when opening an
 ISO-8859-1 XML stream with Turkish as the default locale

improve test name
---
 .../java/org/apache/commons/io/output/XmlStreamWriterTest.java  | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/test/java/org/apache/commons/io/output/XmlStreamWriterTest.java b/src/test/java/org/apache/commons/io/output/XmlStreamWriterTest.java
index 7d3cd1358fe..f7edb8d81a7 100644
--- a/src/test/java/org/apache/commons/io/output/XmlStreamWriterTest.java
+++ b/src/test/java/org/apache/commons/io/output/XmlStreamWriterTest.java
@@ -111,7 +111,7 @@ public void testDefaultEncoding() throws IOException {
     // Turkish language has specific rules to convert dotted and dottless i character.
     @Test
     @SystemDefaults(locale="tr")
-    public void testLowerCaseEncoding_IO_557() throws IOException {
+    public void testLowerCaseEncodingWithTurkishLocale_IO_557() throws IOException {
         checkXmlWriter(TEXT_UNICODE, "utf-8");
         checkXmlWriter(TEXT_LATIN1, "iso-8859-1");
         checkXmlWriter(TEXT_LATIN7, "iso-8859-7");
